project/api/models.py

```diff
@@ -88,10 +97,50 @@ def __str__(self):
 
 class DiplomaticRelation(models.Model):
     """
-    Abstract class for relationships between states, including wars, alliances, overlordships, etc.
-    This would be a whole 'nother project that should probably be primarily populated by Wikipedia scraping
-    Some relations will most likely be controversial, for example the earldoms of Ireland swore allegiance to
-        the English monarchs but in otherwise acted entirely independently, maybe have an autonomy slider
-        or something similar?
+    Defines political and diplomatic interactions between Nations.
     """
-    pass
+    start_date = models.DateField(help_text="When this relation takes effect")
+    end_date = models.DateField(help_text="When this relation ceases to exist")
+    first_party = models.ManyToManyField(Nation, related_name='first_parties')
+    second_party = models.ManyToManyField(Nation, related_name='second_parties')
+    DIPLO_TYPE_CHOICES = (
+        ("A", "Military Alliance"),
+        ("D", "Dual Monarchy"),
+        ("M", "Condominium"),
+        ("T", "Trade League"),
+        ("W", "At War"),
+        ("CP", "Client State - Puppet State"),
+        ("CV", "Client State - Vassal State"),
+        ("CPU", "Client State - Personal Union"),
+        ("CCR", "Client State - Colony - Royal"),
+        ("CCP", "Client State - Colony - Propreitary"),
+        ("CCC", "Client State - Colony - Charter"),
+    )
+    diplo_type = models.TextField(
+        max_length=3,
+        choices=DIPLO_TYPE_CHOICES,
+        default="CC",
```

Comment:
I don't think we should have a default here.

Reply:
Oh yeah, forgot to get rid of that